os-autoinst/openQA
# fullstack: Add context to result panel wait

- Move waits into subtests to make prove work correctly
- Add context to result panel wait
- Enable diagnostics for serialization errors
- Ignore network/Close received console error
- Wait for job element to click on
- Make find_status_text wait or print source

See: https://progress.opensuse.org/issues/98952
